Cargo.toml

```diff
@@ -21,6 +21,7 @@ unicode-bidi-mirroring = "0.3.0"
 unicode-ccc = "0.3.0"
 unicode-properties = { version = "0.1.0", default-features = false, features = ["general-category"] }
 unicode-script = "0.5.2"
+wasmtime = "22.0.0"
```

Comment:
This should at least be optional; I don't want to pull in the entirety of wasmtime and cranelift for a text shaper.

If it were me I would just implement a WASM interpreter inline in rustybuzz. It's not like WASM is that complicated to parse, and it's not like we need WASI IIRC.

Reply:
Yes of course. I would add all the feature flags and such once I am done

Reply:
What I'm saying is that wasmtime is definitely overkill for this use case. I doubt that embedded fonts will need WASI or JIT compilation.